ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,155 +1,169 @@
+{%         if vlan_interfaces[vlan_interface].access_group_out is arista.avd.defined %}
    ip access-group {{ vlan_interfaces[vlan_interface].access_group_out }} out
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_access_group_in is defined and vlan_interfaces[vlan_interface].ipv6_access_group_in is not none %}
+{%         if vlan_interfaces[vlan_interface].ipv6_access_group_in is arista.avd.defined %}
    ipv6 access-group {{ vlan_interfaces[vlan_interface].ipv6_access_group_in }} in
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_access_group_out is defined and vlan_interfaces[vlan_interface].ipv6_access_group_out is not none %}
+{%         if vlan_interfaces[vlan_interface].ipv6_access_group_out is arista.avd.defined %}
    ipv6 access-group {{ vlan_interfaces[vlan_interface].ipv6_access_group_out }} out
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ospf_network_point_to_point is defined and vlan_interfaces[vlan_interface].ospf_network_point_to_point == true %}
+{%         if vlan_interfaces[vlan_interface].ospf_network_point_to_point is arista.avd.defined(true) %}
    ip ospf network point-to-point
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ospf_area is defined and vlan_interfaces[vlan_interface].ospf_area is not none %}
+{%         if vlan_interfaces[vlan_interface].ospf_area is arista.avd.defined %}
    ip ospf area {{ vlan_interfaces[vlan_interface].ospf_area }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ospf_cost is defined and vlan_interfaces[vlan_interface].ospf_cost is not none %}
+{%         if vlan_interfaces[vlan_interface].ospf_cost is arista.avd.defined %}
    ip ospf cost {{ vlan_interfaces[vlan_interface].ospf_cost }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ospf_authentication is defined and vlan_interfaces[vlan_interface].ospf_authentication is not none %}
+{%         if vlan_interfaces[vlan_interface].ospf_authentication is arista.avd.defined %}
 {%           if vlan_interfaces[vlan_interface].ospf_authentication == "simple" %}
    ip ospf authentication
 {%           elif vlan_interfaces[vlan_interface].ospf_authentication == "message-digest" %}
    ip ospf authentication message-digest
 {%           endif %}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ospf_authentication_key is defined and vlan_interfaces[vlan_interface].ospf_authentication_key is not none %}
+{%         if vlan_interfaces[vlan_interface].ospf_authentication_key is arista.avd.defined %}
    ip ospf authentication-key 7 {{ vlan_interfaces[vlan_interface].ospf_authentication_key }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ospf_message_digest_keys is defined and vlan_interfaces[vlan_interface].ospf_message_digest_keys is not none %}
+{%         if vlan_interfaces[vlan_interface].ospf_message_digest_keys is arista.avd.defined %}
```

Comment:
This check is included in natural_sort, so we no longer need to have this if outside the for-loops, as long as we use natural_sort.